pkg/printers/printers.go

```diff
@@ -124,11 +124,25 @@ func GetStandardPrinter(typer runtime.ObjectTyper, encoder runtime.Encoder, deco
 
 		printer = customColumnsPrinter
 
-	case "wide":
-		fallthrough
-	case "":
+	//case "wide", "":
+	//	humanPrintFlags := NewHumanPrintFlags(options.NoHeaders, options.WithNamespace, options.AbsoluteTimestamps)
```

Comment:
Had to remove this case as depending on `pkg/printers/humanprintflags` restores the import cycle between the `printers` package and `printers/internalversion` (printers -> humanprintflags -> internalversion -> printers).